distribution/osbuild-composer.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=OSBuild Composer
+After=multi-user.target
+# Running osbuild-composer without the worker is useless
```

Comment:
Actually, in the future we don't want to require the workers to be local, so I would drop the requriement. But rather pull in the worker in the install section, so that by default enabling composer enables a worker (but you can disable it if you don't want it).